src/data/community-events.json

```
@@ -647,6 +647,15 @@
     "description": "Join the second edition of the event in the center of Venice, to learn more about Blockchain, Technology and with an eye towards art.",
     "startDate": "2024-12-02",
     "endDate": "2024-12-03",
-    "imageUrl": "https://ethvenice.com/images/banner.jpg"
+    "imageUrl": "https://www.ethvenice.com/images/logo.png"
+  },
+  {
+    "title": "ETH Riyadh 2024",
+    "href": "https://ethriyadh.com/",
+    "location": "Riyadh, Saudi Arabia",
+    "description": "ETH Riyadh 2024 unites Middle Eastern developers and blockchain enthusiasts to explore and advance Ethereum technology. Join us for an exciting event that fosters collaboration and innovation in the blockchain ecosystem.",
+    "startDate": "12/1/2024",
+    "endDate": "12/3/2024",
```

dates must follow this convention 

```suggestion
    "startDate": "2024-12-1",
    "endDate": "2024-12-3",
```